toot/api.py

```diff
@@ -361,6 +361,12 @@ def whois(app, user, account):
     return http.get(app, user, f'/api/v1/accounts/{account}').json()
 
 
+def vote(app, user, poll_id, choices: list):
```

Comment:
Since you're adding types, specify what should be in the list. Looking at the API, it's integers so `List[int]`.